WordPress/wordcamp.org
# Application tracker improvements

Improve WordCamp and Meetup application tracker UX. 

Fixes #519, #372 

### Remove Meetup Swag notes metabox

Swag related information belongs to log instead of its own text field, because over time Meetup can get more swag related notes (as we send those out at request regularly). The log is better also because there the note has always a timestamp and trace of the user who made the note.

**NOTE! Before merging this PR to production, all existing swag notes need to be migrated automatically to part of the log.** I can help with tooling for that if needed.

### "Already a meetup" field

Remove "Already a meetup field" from Meetup applications, because same data is visible on the original application and all other fields in the metabox are for tracking the steps for application.

### Move note metabox to the main column and make it first

Sometimes, especially when doing a vetting for application, the deputy needs to write a long note. Having a small text field on sidebar does not make it easy, so move the metabox and text field to main column. Because sometimes there's need to just quickly add a short note, make the metabox as a first in view.

### Improving the "Original application" section

The original application is now displayed as a table and question key is sanitized to more human readable format.

### Screenshots

#### Old note metabox
![](https://i.imgur.com/kRu038e.png)

#### New note metabox
![](https://i.imgur.com/5ceVXhe.png)

#### Old "Original Application" metabox
![](https://i.imgur.com/JoUuCAQ.png)

#### New "Original Application" metabox
![](https://i.imgur.com/5WEZueS.png)

### How to test the changes in this Pull Request:

1. Log in to Central
2. Visit some Meetup application and confirm changes described above
3. Visit some WordCamp application and confirm changes described above
